### ISSUE TITLE
Conflicting import, imported name 'NavArgs' is ambiguous

### ISSUE THREAD

**Issue by RMF09** [raamcosta/compose-destinations#162]
I have two screen composable with destination annotations (PlaceListingScreen and PlaceInfoScreen with arguments), when I added new destination for AddEditPlaceScreen with arguments. This make cause error message **Conflicting import, imported name 'NavArgs' is ambiguous**. I have tried without arguments for AddEditPlaceScreen then work normally. In Previous I use version 1.1.2-beta. the app run normally with arguments of AddEditPlaceScreen but when I update the version to 1.5.11-beta it causing error.

**Error in NavArgsGetter.kt (under build folder generated)**
```
import androidx.compose.material3.ExperimentalMaterial3Api
import androidx.lifecycle.SavedStateHandle
import com.example.medisyclinics.presentation.destinations.AddEditPlaceScreenDestination
import com.example.medisyclinics.presentation.destinations.AddEditPlaceScreenDestination.NavArgs
import com.example.medisyclinics.presentation.destinations.PlaceInfoScreenDestination
import com.example.medisyclinics.presentation.destinations.PlaceInfoScreenDestination.NavArgs

@ExperimentalMaterial3Api
inline fun <reified T> SavedStateHandle.navArgs(): T {
    return navArgs(T::class.java, this)
}

@Suppress("UNCHECKED_CAST")
@ExperimentalMaterial3Api
fun <T> navArgs(argsClass: Class<T>, savedStateHandle: SavedStateHandle): T {
    return when (argsClass) {
		AddEditPlaceScreenDestination.NavArgs::class.java -> AddEditPlaceScreenDestination.argsFrom(savedStateHandle) as T
		PlaceInfoScreenDestination.NavArgs::class.java -> PlaceInfoScreenDestination.argsFrom(savedStateHandle) as T
        else -> error("Class ${argsClass} is not a navigation arguments class!")
    }
}
```
**PlaceInfoScreen.kt**
```@ExperimentalMaterial3Api
@Destination
@Composable
fun PlaceInfoScreen(
    placeId: Int,
    navigator: DestinationsNavigator,
    viewModel: PlaceInfoViewModel = hiltViewModel(),
) {
..... the rest of codes
```
**AddEditPlaceScreen.kt**
```@ExperimentalMaterial3Api
@Destination
@Composable
fun AddEditPlaceScreen(
    isEditMode: Boolean = false,
    viewModel: AddEditPlaceViewModel = hiltViewModel()
) {
... the rest of codes
```

**Comment by RMF09** [comment]
I solved this data class as NavArgs and using params navArgsDelegate in Destination annotations  for AddEditPlaceScreen.  After I read the updated documentation and Sample code application I missed something new . Sorry...
```
data class AddEditPlaceNavArgs(
    val isEditMode: Boolean = false,
    val place: PlaceListing? = null
)

@Destination(navArgsDelegate = AddEditPlaceNavArgs::class)
@Composable
fun AddEditPlaceScreen(
    isEditMode: Boolean = false,
    viewModel: AddEditPlaceViewModel = hiltViewModel()
) {
... the rest of codes
```

**Comment by RMF09** [comment]
I solved this data class as NavArgs and using params navArgsDelegate in Destination annotations  for AddEditPlaceScreen.  After I read the updated documentation and Sample code application I missed something new . Sorry...
```
data class AddEditPlaceNavArgs(
    val isEditMode: Boolean = false,
    val place: PlaceListing? = null
)

@Destination(navArgsDelegate = AddEditPlaceNavArgs::class)
@Composable
fun AddEditPlaceScreen(
    isEditMode: Boolean = false,
    viewModel: AddEditPlaceViewModel = hiltViewModel()
) {
... the rest of codes
```

**Comment by raamcosta** [comment]
@RMF09 this is still a bug, don't close before I look at it 😄 

No problem though, someone else has reported it and a new version is going to be released soon. Update to it! (it will be 1.5.12-beta and it should be available in the next hour or so).

**Comment by raamcosta** [comment]
New version available 1.5.12-beta!

You can update and revert the workaround you did 🙂

**Comment by RMF09** [comment]
Oke thank you @raamcosta 👍
